src/App.js

```diff
@@ -1,23 +1,186 @@
+          {gameStarted === 0 ? <div className='questionDiv'><p className='introText'> Greetings, traveler! We seem to have run out of fuel on our voyage and need your help. Answer trivia questions correctly and help us earn the credits to re-fuel at five friendly planets. Get us to 500 and we'll be able to make the journey home!</p></div> : 
+            
+          <div className='questionDiv'>
+            <p dangerouslySetInnerHTML={{ __html: question }}></p>
+            <form onSubmit={handleSubmit} className='form'>
+              <label htmlFor='trueOption'>True</label>
+              <input type="radio" id='trueOption' name='quizBoolean' value='True' onChange={handleChange} />
+              <label htmlFor='falseOption'>False</label>
+              <input type="radio" id='falseOption' name='quizBoolean' value='False' onChange={handleChange} />
+              <br></br>
+              <button type='submit' disabled={formSubmitted} className='answerButton'>Answer</button>
+            </form>
+
+          </div>
+            }
+          {gameStarted === 0 ? null : <DisplayBox />}
+          
+          <div className='scoreDiv'>
+            <p>
+              SCORE
+              <br></br>
+              {score}/500
+            </p>
+          </div>
+        </div>
+          
+        {score < 500 ? null : <div className='finalMessage'>Thanks for playing! Enjoy your return journey or keep working on your trivia skills!</div>}
+        <div className="planetDiv">
+
+          <div className={`image-container ${score >= 100 ? 'animate1' : ''}`}>
+            <img src={image100}
+              style={{
+                display: score >= 100 && score < 200 ? "block" : "none",
+
+              }}
+              alt="Planet 1" />
+          </div>
+          <div className={`image-container ${score >= 200 ? 'animate1' : ''}`}>
+            <img src={image200}
+              style={{
+                display: score >= 200 ? "block" : "none",
+
+              }}
+              alt="Planet 2" />
+          </div>
+          <div className={`image-container ${score >= 300 ? 'animate1' : ''}`}>
+            <img src={image300}
+              style={{
+                display: score >= 300 ? "block" : "none",
+
+              }}
+              alt="Planet 3" />
+          </div>
+          <div className={`image-container ${score >= 400 ? 'animate1' : ''}`}>
+            <img src={image400}
+              style={{
+                display: score >= 400 ? "block" : "none",
+
+              }}
+              alt="Planet 4" />
+          </div>
+          <div className={`image-container ${score >= 500 ? 'animate1' : ''}`}>
+            <img src={image500}
+              style={{
+                display: score >= 500 ? "block" : "none",
+              }}
+              alt="Planet 5" />
+          </div>
+        </div>
+
+
```

Comment:
again, love the organization and cleanliness of your code! And LOVE the app, its super impressive and I am so inspired by it! Just keep an eye out for any commented out code/console logs and make sure you're removing those before submitting :)